packages/peregrine/lib/talons/RootComponents/Category/useCategoryContent.js

```diff
@@ -58,6 +67,18 @@ export const useCategoryContent = props => {
         }
     }, [categoryId, getFilters]);
 
+    useEffect(() => {
```

Comment:
You can combine this effect on the previous one, there's no need to declare a new one.

Reply:
I've seen this in other components, thought it would be best practice to separate in two different `useEffect`s since they call different queries.

Reply:
Ok, seems like the correct way.